drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/columns/control/DeleteColumnManagementAnchorWidget.java

```diff
@@ -16,25 +16,34 @@
 
 package org.drools.workbench.screens.guided.dtable.client.widget.table.columns.control;
 
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
 import com.google.gwt.user.client.Command;
 import com.google.gwt.user.client.ui.Anchor;
 import org.drools.workbench.screens.guided.dtable.client.resources.i18n.GuidedDecisionTableConstants;
-import org.uberfire.ext.widgets.common.client.common.popups.YesNoCancelPopup;
+import org.uberfire.ext.editor.commons.client.file.popups.DeletePopUpPresenter;
 
+@Dependent
 public class DeleteColumnManagementAnchorWidget extends Anchor {
 
-    public DeleteColumnManagementAnchorWidget(final String columnHeader,
-                                              final Command afterDeleteConfirmedCommand) {
+    private Command afterDeleteConfirmedCommand;
+
+    @Inject
+    private DeletePopUpPresenter deletePopUpPresenter;
+
+    public DeleteColumnManagementAnchorWidget() {
         super();
         setText(GuidedDecisionTableConstants.INSTANCE.Delete());
         setTitle(GuidedDecisionTableConstants.INSTANCE.DeleteThisColumn());
         addClickHandler((clickEvent) -> {
-            final String confirmMessage = GuidedDecisionTableConstants.INSTANCE.DeleteColumnWarning(columnHeader);
-            YesNoCancelPopup.newYesNoCancelPopup(GuidedDecisionTableConstants.INSTANCE.Delete(),
-                                                 confirmMessage,
-                                                 () -> afterDeleteConfirmedCommand.execute(),
-                                                 null,
-                                                 null).show();
+            deletePopUpPresenter.getToggleCommentPresenter().getViewElement().setAttribute("hidden",
```

Comment:
@jomarko I don't think it's going to be quite so easy :-( IMO this is really an abuse of the ```DeletePopupPresenter``` as we should not be tweaking it's visual appearance from outside the class (adding a ```enableComment(boolean)``` is probably quite acceptable)... but that also brings me onto...